CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Bug where an empty destination list in a device's backup set broke creation of DeviceSettings objects for that device.
 
+### Added
+
+- Added new exception `Py42CloudAliasLimitExceeded` to throw if `add_cloud_alias` throws `400` and body contains reason
+`Cloud usernames must be less than or equal to 2`.
```

Comment:
Adjust this to reflect the fact that `2` is no longer checked.

Reply:
Changed.